plugins/PrivacyManager/vue/src/ManageGdpr/ManageGdpr.vue

```diff
@@ -306,7 +308,7 @@ export default defineComponent({
         id: 'all',
         name: translate('UsersManager_AllWebsites'),
       },
-      segment_filter: 'userId==',
+      segment_filter: 'visitId==',
```

Comment:
Using `visitId` by default ensure that the initially choosen segment is really available. It may get update anyway, after the request to check if the profile is enabled finishes.